omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1883 @@
+********************
+Version requirements
+********************
+
+Proposal for revised minimum version requirements
+-------------------------------------------------
+
+Minimum software versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a proposal to the wider community to solicit their
```

Comment:
I'd think the PR and/or the emails to ome-users/ome-devel are the proposal and that this document should be the current status of that "version plan" (or whatever), keeping the request for feedback. If that doesn't fit with the more general intent, perhaps under `contributing/` would be better.